Source/Scene/Model.js

```diff
@@ -1186,8 +1187,21 @@ define([
         var cacheKey = defaultValue(options.cacheKey, getAbsoluteUri(url));
 
         options = clone(options);
-        options.basePath = getBaseUri(url, true);
-        options.cacheKey = cacheKey;
+        var providedBasePath = options.basePath;
+        if (!defined(options.basePath)) {
+            options.basePath = getBaseUri(url, true);
+        }
+        if (!defined(options.cacheKey)) {
+            if (defined(providedBasePath)) {
+                options.cacheKey = cacheKey + options.basePath;
+            }
+            else {
```

Comment:
Change this and below to `} else {`

Our code style convention for if/else blocks is
``` javascript
if (...) {
    ...
} else if (...) {
    ...
} else {
    ...
}
```